assets/scripts/evolution/shuffle_gallery.js

```diff
@@ -0,0 +1,61 @@
+export default function() {
+
+	/******** Shuffle Gallery Component ********/
+
+	var links = document.getElementsByClassName("evo_c-shuffle-gallery__link");
```

Comment:
Hey @monalighosh,

Please check for the existence of elements in the DOM before working with them. For example, at line 5, `links` is referring to some elements in the DOM, should it turn out that `links` is `undefined`, your code will break.

You could do:
```js
if (links) { 
//code
}

See this section on [writing safe JS code](https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#writing-safe-javascript-code) for more details.